scanner/src/routes/+page.svelte

```diff
@@ -3,6 +3,12 @@
   import { BarcodeScanner } from "@capacitor-community/barcode-scanner"
   import { Buffer } from "buffer"
   import { decode as decodeBase45 } from "base45"
+  import qrcode from "$assets/qrcode.png"
+  import question from "$assets/question.svg"
+  import logo from "$assets/logo.svg"
+  import camera from "$assets/camera.svg"
+  import subtract from "$assets/subtract.png"
```

Comment:
Changing the import to say `substract.png` does not rename the file from `Substract.png`, now it crashes for me on a case-sensitive filesystem.